src/space.ts

```diff
@@ -0,0 +1,217 @@
+// Deta Space CLI Spec https://github.com/deta/space-cli
+// at latest commit: https://github.com/deta/space-cli/commit/aa93f1476e56268fab0c4444db1e6326feb8f59f
+// using https://github.com/withfig/autocomplete-tools/tree/main/integrations/cobra
+const completionSpec: Fig.Spec = {
+  name: "space",
+  description: "Deta Space CLI for mananging Deta Space projects",
+  subcommands: [
+    {
+      name: "completion",
+      description: "Generate the autocompletion script for the specified shell",
+      subcommands: [
+        {
+          name: "bash",
+          description: "Generate the autocompletion script for bash",
+          options: [
+            {
+              name: "--no-descriptions",
+              description: "Disable completion descriptions",
+            },
+          ],
+        },
+        {
+          name: "fish",
+          description: "Generate the autocompletion script for fish",
+          options: [
+            {
+              name: "--no-descriptions",
+              description: "Disable completion descriptions",
+            },
+          ],
+        },
+        {
+          name: "powershell",
+          description: "Generate the autocompletion script for powershell",
+          options: [
+            {
+              name: "--no-descriptions",
+              description: "Disable completion descriptions",
+            },
+          ],
+        },
+        {
+          name: "zsh",
+          description: "Generate the autocompletion script for zsh",
+          options: [
+            {
+              name: "--no-descriptions",
+              description: "Disable completion descriptions",
+            },
+          ],
+        },
+      ],
+    },
+    {
+      name: "link",
+      description: "Link code to project",
+      options: [
+        {
+          name: ["--dir", "-d"],
+          description: "Src of project to link",
+          args: { name: "dir", default: "./" },
```

Comment:
@BogDAAAMN this can eventually be configured via cobra so that it gets autogenerated when the integration is run.
```suggestion
          args: { name: "dir", default: "./", template: "folders" },
```